SnooksHarvard.csl

```diff
@@ -0,0 +1,188 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only">
+  <info>
+    <title> Snooks Australia Harvard (author-date)</title>
+    <id>http://www.zotero.org/styles/SnooksHarvard</id>
+    <link href="/http[s]?:\/\/www.zotero.org\/styles\/SnooksHarvard/" rel="self"/>
+    <link href="http://www.australia.gov.au/publications/style-manual" rel="documentation"/>
+    <link href="http://www.zotero.org/styles/harvard1" rel="template"/>
+    <author>
+      <name>jstedman</name>
+      <email>jessica.stedman@gmail.com</email>
+    </author>
+    <category citation-format="author-date"/>
+    <category field="generic-base"/>
+    <summary>Snooks style manual Australia Harvard author-date style</summary>
+    <updated>2014-09-10T06:18:55+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="en">
+         <terms>
+	<term name="edition" form="short">edn</term>
+</terms>  
+</locale>
+ <macro name="editor"> 
+    <names variable="editor" delimiter=" " prefix="">
+      <name and="symbol" initialize-with="" delimiter=", "/>
+      <label form="short" prefix=" (" suffix=")" text-case="lowercase"/>
+    </names>
+  </macro>
+  <macro name="anon">
+    <text macro="title"/>
+  </macro>
+  <macro name="author">
+    <names variable="author">
+      <name name-as-sort-order="all" and="symbol" sort-separator=", "  initialize-with="" delimiter-precedes-last="never" delimiter=", "/>
+      <label form="short" prefix=" (" suffix=")" text-case="lowercase"/>
+      <substitute>
+        <names variable="editor"/>
+        <text macro="anon"/>
+      </substitute>
+    </names>
+  </macro>
+  <macro name="author-short">
+    <names variable="author">
+      <name form="short" and="symbol" delimiter=" " delimiter-precedes-last="never"/>
+      <substitute>
+        <names variable="editor"/>
+        <names variable="translator"/>
+        <text macro="anon"/>
+      </substitute>
+    </names>
+  </macro>
+  <macro name="access">
+    <choose>
+      <if variable="URL">
+	  <group suffix=" ">
+          <text value="viewed" text-case="lowercase" suffix=" "/>
+          <date variable="accessed">
+            <date-part name="month" suffix=" "/>
+            <date-part name="day" suffix=" "/>
+            <date-part name="year"/>
+          </date>
+        </group>
+	<text variable="URL" prefix="&lt;" suffix="&gt;"/>
+      </if>
+    </choose>
+  </macro>
+  <macro name="title">
+    <choose>
+      <if type="bill book graphic legal_case legislation motion_picture report song thesis webpage" match="any">
+        <text variable="title" font-style="italic"/>
+      </if>
+      <else>
+        <text variable="title" text-case="title" prefix="'" suffix="'"/>
```

Comment:
Adding `default-locale="en-GB"` (as @adam3smith suggested earlier) should automatically switch the style to using single quotes, as long as you're using `quotes="true"`.